css-grid-1/implicit-grids/grid-layout-auto-columns-rows.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>CSS Grid Layout Test: auto columns/rows</title>
+        <link rel="help" href="http://www.w3.org/TR/css-grid-1/#auto-tracks">
+        <link rel="match" href="../reference/grid-layout-auto-columns-rows-reference.html">
+        <link rel="author" title="jxs" href="mailto:hello@jxs.pt"/>
+        <meta name="assert" content="the layout should behave the same as reference.">
```

Comment:
The "assert" tag should explain something else that is not covered by the title.
The fact that we've to be the same than the reference, it's already known.
So in this particular simple example I won't simply remove the "assert" tag.